zio-http/src/main/scala/zio/http/package.scala

```diff
@@ -20,4 +20,6 @@ package object http extends PathSyntax with RequestSyntax with RouteDecoderModul
 
   object HeaderNames  extends headers.HeaderNames
   object HeaderValues extends headers.HeaderValues
+
+  val failNoStacktrace: ZIO[Any, None.type, Nothing] = ZIO.refailCause(Cause.fail(None))
```

Comment:
I'd make this private to the http package to avoid cluttering the user namespace with implementation details.
```suggestion
  private[http] val failNoStacktrace: ZIO[Any, None.type, Nothing] = ZIO.refailCause(Cause.fail(None))
```